base/cmd.jl

```diff
@@ -16,6 +16,8 @@ struct Cmd <: AbstractCmd
     cpus::Union{Nothing,Vector{UInt16}}
     Cmd(exec::Vector{String}) =
         new(exec, false, 0x00, nothing, "", nothing)
+    Cmd(exec::Vector{AbstractString}) =
+        new(string.(exec), false, 0x00, nothing, "", nothing)
```

Comment:
I wonder if this would also work

```julia
Cmd(exec::Vector{AbstractString}) = Cmd(string.(exec))
```

to avoid duplicating the defaults.

Reply:
Was wondering if that would be okay in an inner constructor, makes sense though.